core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7615,6 +7636,15 @@ public static final class Name {
     public static final String TABLE_LOAD_DEFAULT_REPLICATION =
         "alluxio.table.load.default.replication";
 
+    public static final String HADOOP_SECURITY_AUTHENTICATION =
+            "alluxio.hadoop.security.authentication";
```

Comment:
is the indentation correct here?
```suggestion
            "alluxio.hadoop.security.authentication";
```